utilities/manifests/onnx.py

```diff
@@ -2,22 +2,26 @@
+                }],
+                "query_output": r'{"model_name":"$model_name","model_version":"1","outputs":\[\{"name":"Plus214_Output_0","shape":\[1,10\],"datatype":"FP32","data":\[.*\]}]}',
+                "use_regex": True
+            },
+            "infer-mnist": {
+                "query_input": "@utilities/manifests/openvino/mnist-input.json",
+                "query_output": r'{"model_name":"mnist-model__isvc-[0-9a-z]+","model_version":"1","outputs":\[{"name":"Plus214_Output_0","datatype":"FP32","shape":\[1,10\],"data":\[.*\]}\]}',
+                "use_regex": True
+            },
+            "graph": {
+                "query_input": "@utilities/manifests/openvino/dog-input-tensor.json",
+                "query_output": r'{"model_name":"dog-breed-classifier","model_version":"1","outputs":\[{"name":"495","shape":\[1,133\],"datatype":"FP32","data":\[.*\]}\]}',
```

Comment:
`dog-breed-classifier` should be replaced with $model_name

Reply:
I changed it, but I haven't tried the change. Let me confirm it works as intended.

Reply:
I can confirm it works.